docs/supported_platforms.md

```diff
@@ -2,7 +2,7 @@
 
 ## Officially supported platforms
 
-  + Ubuntu 18.04 and 20.04
+  + Ubuntu 18.04, 20.04, 21.10, 22.04
```

Comment:
I think we had decided to not officially support 21.10 since its lifespan is so short and we only wanted to support LTS releases. I think we decided that stopping support for a platform would be considered a breaking change, so I think it would be better to leave it as unsupported (plus it leaves more room in the CI for future LTS platforms that we'll also need to support).